dependency/dependency_test.go

```diff
@@ -241,6 +236,7 @@ func (c *ClientSet) createConsulTestResources() error {
 		if err := testClients.createConsulPeerings(tenancy); err != nil {
 			return err
 		}
+		time.Sleep(200 * time.Millisecond)
```

Comment:
Is this needed? The creation of resources should be in sync right?

Reply:
this is to make sure that consul reaches a state of entropy.
the falky tests were due to wrong way of creating resource but the consul servers were not reaching a state of entropy and thus were not evicting the resources sometimes.

Reply:
If you changed all the registrations so they're not to the local agent but to a fake node then agent entropy won't matter so there should be no need for this sleep

Reply:
noted, I had it in place so that if someone accidentally writes to the local agent, the tests would keep on failing, without getting successful intermittently.

What do you suggest?